src/prop/prop_proof_manager.h

```diff
@@ -169,6 +178,38 @@ class PropPfManager : protected EnvObj
   std::vector<Node> getInputClauses();
   /** Retrieve the clauses derived from lemmas */
   std::vector<Node> getLemmaClauses();
+  /** Return lemmas used in the SAT proof. */
```

Comment:
amend this to account for the fact that you only use the SAT proof if there is one that allows you to get the lemmas, otherwise this will be retrieved some other way.